enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -1721,6 +1721,9 @@ void clearFunctionAttributes(Function *f) {
     f->removeRetAttr(Attribute::Alignment);
   }
   Attribute::AttrKind attrs[] = {
+#if LLVM_VERSION_MAJOR >= 20
+    Attribute::Range,
```

Comment:
can you check which version of llvm introduced the range attr (and confirm it makes your number here and in the test)

Reply:
[my lazy trick for this is removing the range, and seeing what the first version which compiles]

Reply:
updated, 18 failed, 19 worked.